scrypto-unit/src/test_runner.rs

```diff
@@ -235,24 +235,38 @@ impl CustomGenesis {
     }
 }
 
-pub struct TestRunnerBuilder {
+pub trait Database:
```

Comment:
Perhaps change name to `TestDatabase` as it's the fact that it's a test that we need all of the listed traits.

Reply:
I always welcome a good naming suggestion 👍